File: src/Libro.py
import random

class Libro:
    def __init__(self, nombre, autor):
        self.nombre = nombre
        self.id = random.randint(1, 1000)
        self.autor = autor
        self.es_rentado = False
        self.es_comprado = False

    def get_nombre(self):
        return self.nombre

    def get_autor(self):
        return self.autor

    def get_es_rentado(self):
        return self.es_rentado

    def set_es_rentado(self, rentado):
        self.es_rentado = rentado

    def get_es_comprado(self):
        return self.es_comprado

    def set_es_comprado(self, comprado):
        self.es_comprado = comprado

File: src/Libro.java
import java.util.Random;

public class Libro {
    private String nombre;
    private int id;
    private String autor;
    private  boolean EsRentado;
    private boolean EsComprado;
    Random ran=new Random();

    public Libro(String nombre,String autor){
        this.nombre=nombre;
        this.id= ran.nextInt(1000)+1;
        this.autor=autor;
        this.EsRentado=false;
        this.EsComprado=false;
    }

    public boolean getEsComprado() {
        return EsComprado;
    }

    public void setEsComprado(boolean esComprado) {
        this.EsComprado = esComprado;
    }

    public boolean getEsRentado() {
        return EsRentado;
    }

    public void setEsRentado(boolean esRentado) {
        this.EsRentado = esRentado;
    }

    public String getNombre() {
        return nombre;
    }

    public String getAutor() {
        return autor;
    }
}
